src/faaslet/Faaslet.cpp

```diff
@@ -31,7 +31,7 @@ void preloadPythonRuntime()
 {
     auto logger = faabric::util::getLogger();
 
-    auto conf = faabric::util::getSystemConfig();
+    auto& conf = faabric::util::getSystemConfig();
```

Comment:
I noticed a lot of places we were using `auto` where we should be using `auto&` to avoid a copy.

Reply:
True, I'll also be on the lookout for this